src/Theta/Transaction.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Transaction.h"
+
+#include "../Ethereum/RLP.h"
+
+using namespace TW;
+using namespace TW::Theta;
+using RLP = Ethereum::RLP;
+
+Data encode(const Coins& coins) noexcept {
+    auto encoded = Data();
+    append(encoded, RLP::encode(coins.thetaWei));
+    append(encoded, RLP::encode(coins.tfuelWei));
+    return RLP::encodeList(encoded);
+}
+
+Data encode(const TxInput& input) noexcept {
+    auto encoded = Data();
+    append(encoded, RLP::encode(input.address.bytes));
+    append(encoded, encode(input.coins));
+    append(encoded, RLP::encode(input.sequence));
+    append(encoded, RLP::encode(input.signature));
+    return RLP::encodeList(encoded);
+}
+
+Data encode(const std::vector<TxInput>& inputs) noexcept {
+    auto encoded = Data();
+    for (const auto& input : inputs)
+    {
```

Comment:
could you please run `clang-format` and follow existing code style?

Reply:
Yep let me do. Thank you.